BloblStoreTest failed on Windows

See log below:
```
com.couchbase.lite.BlobStoreTest > testBasic STANDARD_ERROR
    Oct 01, 2015 4:45:10 AM com.couchbase.lite.util.SystemLogger d
    FINE: Database: Starting Manager version: devbuild
    Oct 01, 2015 4:45:10 AM com.couchbase.lite.util.SystemLogger w
    WARNING: Database: mustExist is true and db (cblite-test) does not exist
    Oct 01, 2015 4:45:10 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Opening com.couchbase.lite.Database[C:\Users\pasin\Git\couchbase-lite-java\data\data\com.couchbase.
lite.test\files\test\cblite-test.cblite2]
    Oct 01, 2015 4:45:10 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: AndroidSQLCipherStorageEngine{database=33fd6994}: Opened Android sqlite db
    Oct 01, 2015 4:45:10 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Begin transaction (level 0)
    Oct 01, 2015 4:45:10 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: calls optimizeSQLIndexes()
    Oct 01, 2015 4:45:10 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Committing transaction (level 0)
    Oct 01, 2015 4:45:13 AM com.couchbase.lite.util.SystemLogger v
    FINER: LiteTestCaseWithDB: tearDown
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger d
    FINE: Database: Closing com.couchbase.lite.Manager@d804ac3
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger d
    FINE: Database: Closed com.couchbase.lite.Manager@d804ac3
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger e
    SEVERE: LiteTestCase: DURATION: testBasic: 3 sec - [SLOW]
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger e
    SEVERE: LiteTestCaseWithDB: DURATION: testBasic: 3 sec
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger v
    FINER: LiteTestCaseWithDB: setUp
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger d
    FINE: Database: Starting Manager version: devbuild
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Opening com.couchbase.lite.Database[C:\Users\pasin\Git\couchbase-lite-java\data\data\com.couchbase.
lite.test\files\test\cblite-test.cblite2]
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: AndroidSQLCipherStorageEngine{database=7fe0813a}: Opened Android sqlite db
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Begin transaction (level 0)
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Committing transaction (level 0)
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Opening com.couchbase.lite.Database[C:\Users\pasin\Git\couchbase-lite-java\data\data\com.couchbase.
lite.test\files\test\cblite-test.cblite2]
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: AndroidSQLCipherStorageEngine{database=50211475}: Opened Android sqlite db
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Begin transaction (level 0)
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: calls optimizeSQLIndexes()
    Oct 01, 2015 4:45:14 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Committing transaction (level 0)
    Oct 01, 2015 4:45:15 AM com.couchbase.lite.util.SystemLogger i
    INFO: LiteTestCaseWithDB: ---- Now enabling attachment encryption ----
    Oct 01, 2015 4:45:15 AM com.couchbase.lite.util.SystemLogger e
    SEVERE: Database: BlobStore: Blob store was corrupted. Encryption marker file was not found

com.couchbase.lite.BlobStoreTest > testBasic FAILED
    com.couchbase.lite.CouchbaseLiteException
        at com.couchbase.lite.BlobStore.verifyExistingStore(BlobStore.java:112)
        at com.couchbase.lite.BlobStore.<init>(BlobStore.java:69)
        at com.couchbase.lite.BlobStoreTest.setUp(BlobStoreTest.java:59)
        at junit.framework.TestCase.runBare(TestCase.java:139)
        at com.couchbase.lite.LiteTestCase.runBare(LiteTestCase.java:20)
        at com.couchbase.lite.LiteTestCaseWithDB.runBare(LiteTestCaseWithDB.java:77)
        at com.couchbase.lite.BlobStoreTest.runBare(BlobStoreTest.java:46)

com.couchbase.lite.BlobStoreTest > testBlobStoreWriter STANDARD_ERROR
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: LiteTestCaseWithDB: setUp
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger d
    FINE: Database: Starting Manager version: devbuild
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Opening com.couchbase.lite.Database[C:\Users\pasin\Git\couchbase-lite-java\data\data\com.couchbase.
lite.test\files\test\cblite-test.cblite2]
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: AndroidSQLCipherStorageEngine{database=79e240f0}: Opened Android sqlite db
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Begin transaction (level 0)
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Committing transaction (level 0)

com.couchbase.lite.BlobStoreTest > testBlobStoreWriter FAILED
    com.couchbase.lite.CouchbaseLiteException: Was not able to delete the database directory
        at com.couchbase.lite.Database.delete(Database.java:308)
        at com.couchbase.lite.LiteTestCaseWithDB.ensureEmptyDatabase(LiteTestCaseWithDB.java:179)
        at com.couchbase.lite.LiteTestCaseWithDB.startDatabase(LiteTestCaseWithDB.java:166)
        at com.couchbase.lite.LiteTestCaseWithDB.setUp(LiteTestCaseWithDB.java:97)
        at com.couchbase.lite.BlobStoreTest.setUp(BlobStoreTest.java:51)
        at junit.framework.TestCase.runBare(TestCase.java:139)
        at com.couchbase.lite.LiteTestCase.runBare(LiteTestCase.java:20)
        at com.couchbase.lite.LiteTestCaseWithDB.runBare(LiteTestCaseWithDB.java:77)
        at com.couchbase.lite.BlobStoreTest.runBare(BlobStoreTest.java:43)

com.couchbase.lite.BlobStoreTest > testBlobStoreWriterBasicOperation STANDARD_ERROR
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: LiteTestCaseWithDB: setUp
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger d
    FINE: Database: Starting Manager version: devbuild
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Opening com.couchbase.lite.Database[C:\Users\pasin\Git\couchbase-lite-java\data\data\com.couchbase.
lite.test\files\test\cblite-test.cblite2]
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: AndroidSQLCipherStorageEngine{database=51d52040}: Opened Android sqlite db
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Begin transaction (level 0)
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Committing transaction (level 0)

com.couchbase.lite.BlobStoreTest > testBlobStoreWriterBasicOperation FAILED
    com.couchbase.lite.CouchbaseLiteException: Was not able to delete the database directory
        at com.couchbase.lite.Database.delete(Database.java:308)
        at com.couchbase.lite.LiteTestCaseWithDB.ensureEmptyDatabase(LiteTestCaseWithDB.java:179)
        at com.couchbase.lite.LiteTestCaseWithDB.startDatabase(LiteTestCaseWithDB.java:166)
        at com.couchbase.lite.LiteTestCaseWithDB.setUp(LiteTestCaseWithDB.java:97)
        at com.couchbase.lite.BlobStoreTest.setUp(BlobStoreTest.java:51)
        at junit.framework.TestCase.runBare(TestCase.java:139)
        at com.couchbase.lite.LiteTestCase.runBare(LiteTestCase.java:20)
        at com.couchbase.lite.LiteTestCaseWithDB.runBare(LiteTestCaseWithDB.java:77)
        at com.couchbase.lite.BlobStoreTest.runBare(BlobStoreTest.java:43)

com.couchbase.lite.BlobStoreTest > testReopen STANDARD_ERROR
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: LiteTestCaseWithDB: setUp
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger d
    FINE: Database: Starting Manager version: devbuild
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Opening com.couchbase.lite.Database[C:\Users\pasin\Git\couchbase-lite-java\data\data\com.couchbase.
lite.test\files\test\cblite-test.cblite2]
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: AndroidSQLCipherStorageEngine{database=16d47954}: Opened Android sqlite db
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Begin transaction (level 0)
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Committing transaction (level 0)

com.couchbase.lite.BlobStoreTest > testReopen FAILED
    com.couchbase.lite.CouchbaseLiteException: Was not able to delete the database directory
        at com.couchbase.lite.Database.delete(Database.java:308)
        at com.couchbase.lite.LiteTestCaseWithDB.ensureEmptyDatabase(LiteTestCaseWithDB.java:179)
        at com.couchbase.lite.LiteTestCaseWithDB.startDatabase(LiteTestCaseWithDB.java:166)
        at com.couchbase.lite.LiteTestCaseWithDB.setUp(LiteTestCaseWithDB.java:97)
        at com.couchbase.lite.BlobStoreTest.setUp(BlobStoreTest.java:51)
        at junit.framework.TestCase.runBare(TestCase.java:139)
        at com.couchbase.lite.LiteTestCase.runBare(LiteTestCase.java:20)
        at com.couchbase.lite.LiteTestCaseWithDB.runBare(LiteTestCaseWithDB.java:77)
        at com.couchbase.lite.BlobStoreTest.runBare(BlobStoreTest.java:43)

com.couchbase.lite.BlobStoreTest > testBlobStoreWriterForBody STANDARD_ERROR
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: LiteTestCaseWithDB: setUp
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger d
    FINE: Database: Starting Manager version: devbuild
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Opening com.couchbase.lite.Database[C:\Users\pasin\Git\couchbase-lite-java\data\data\com.couchbase.
lite.test\files\test\cblite-test.cblite2]
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: AndroidSQLCipherStorageEngine{database=18dd8196}: Opened Android sqlite db
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Begin transaction (level 0)
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Committing transaction (level 0)

com.couchbase.lite.BlobStoreTest > testBlobStoreWriterForBody FAILED
    com.couchbase.lite.CouchbaseLiteException: Was not able to delete the database directory
        at com.couchbase.lite.Database.delete(Database.java:308)
        at com.couchbase.lite.LiteTestCaseWithDB.ensureEmptyDatabase(LiteTestCaseWithDB.java:179)
        at com.couchbase.lite.LiteTestCaseWithDB.startDatabase(LiteTestCaseWithDB.java:166)
        at com.couchbase.lite.LiteTestCaseWithDB.setUp(LiteTestCaseWithDB.java:97)
        at com.couchbase.lite.BlobStoreTest.setUp(BlobStoreTest.java:51)
        at junit.framework.TestCase.runBare(TestCase.java:139)
        at com.couchbase.lite.LiteTestCase.runBare(LiteTestCase.java:20)
        at com.couchbase.lite.LiteTestCaseWithDB.runBare(LiteTestCaseWithDB.java:77)
        at com.couchbase.lite.BlobStoreTest.runBare(BlobStoreTest.java:43)

com.couchbase.lite.BlobStoreTest > testLiteTestCaseSetupProperly STANDARD_ERROR
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: LiteTestCaseWithDB: setUp
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger d
    FINE: Database: Starting Manager version: devbuild
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Opening com.couchbase.lite.Database[C:\Users\pasin\Git\couchbase-lite-java\data\data\com.couchbase.
lite.test\files\test\cblite-test.cblite2]
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: AndroidSQLCipherStorageEngine{database=5dd6341a}: Opened Android sqlite db
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Begin transaction (level 0)
    Oct 01, 2015 4:45:16 AM com.couchbase.lite.util.SystemLogger v
    FINER: Database: Test worker Committing transaction (level 0)

com.couchbase.lite.BlobStoreTest > testLiteTestCaseSetupProperly FAILED
    com.couchbase.lite.CouchbaseLiteException: Was not able to delete the database directory
        at com.couchbase.lite.Database.delete(Database.java:308)
        at com.couchbase.lite.LiteTestCaseWithDB.ensureEmptyDatabase(LiteTestCaseWithDB.java:179)
        at com.couchbase.lite.LiteTestCaseWithDB.startDatabase(LiteTestCaseWithDB.java:166)
        at com.couchbase.lite.LiteTestCaseWithDB.setUp(LiteTestCaseWithDB.java:97)
        at com.couchbase.lite.BlobStoreTest.setUp(BlobStoreTest.java:51)
        at junit.framework.TestCase.runBare(TestCase.java:139)
        at com.couchbase.lite.LiteTestCase.runBare(LiteTestCase.java:20)
        at com.couchbase.lite.LiteTestCaseWithDB.runBare(LiteTestCaseWithDB.java:77)
        at com.couchbase.lite.BlobStoreTest.runBare(BlobStoreTest.java:43)
Gradle Test Executor 1 finished executing tests.

6 tests completed, 6 failed
Finished generating test XML results (0.171 secs) into: C:\Users\pasin\Git\couchbase-lite-java\build\test-results
Generating HTML test report...
Finished generating test html results (0.094 secs) into: C:\Users\pasin\Git\couchbase-lite-java\build\reports\tests
:test FAILED
:test (Thread[main,5,main]) completed. Took 12.363 secs.

FAILURE: Build failed with an exception.
```

--- a/src/main/java/com/couchbase/lite/BlobStore.java
+++ b/src/main/java/com/couchbase/lite/BlobStore.java
@@ -401,7 +401,6 @@ public class BlobStore {
                 );
             }
         }
-
         return filename;
     }
 
@@ -555,30 +554,33 @@ public class BlobStore {
     }
 
     private static byte[] getBytesFromFile(File file) throws IOException {
-        InputStream is = new FileInputStream(file);
+        InputStream is = null;
+        try {
+            is = new FileInputStream(file);
 
-        // Get the size of the file
-        long length = file.length();
+            // Get the size of the file
+            long length = file.length();
 
-        // Create the byte array to hold the data
-        byte[] bytes = new byte[(int) length];
+            // Create the byte array to hold the data
+            byte[] bytes = new byte[(int) length];
 
-        // Read in the bytes
-        int offset = 0;
-        int numRead = 0;
-        while (offset < bytes.length
-                && (numRead = is.read(bytes, offset, bytes.length - offset)) >= 0) {
-            offset += numRead;
+            // Read in the bytes
+            int offset = 0;
+            int numRead = 0;
+            while (offset < bytes.length
+                    && (numRead = is.read(bytes, offset, bytes.length - offset)) >= 0) {
+                offset += numRead;
+            }
+
+            // Ensure all the bytes have been read in
+            if (offset < bytes.length) {
+                throw new IOException("Could not completely read file " + file.getName());
+            }
+            return bytes;
+        } finally {
+             if (is != null)
+                 is.close();
         }
-
-        // Ensure all the bytes have been read in
-        if (offset < bytes.length) {
-            throw new IOException("Could not completely read file " + file.getName());
-        }
-
-        // Close the input stream and return bytes
-        is.close();
-        return bytes;
     }
 
     public Set<BlobKey> allKeys() {

